django/db/backends/postgresql/features.py

```diff
@@ -93,6 +93,13 @@ class DatabaseFeatures(BaseDatabaseFeatures):
         },
     }
 
+    # Indicates if the database backend supports conditional updates in bulk
+    # operations. True for backends that allow syntax like
+    # "INSERT INTO ... ON CONFLICT (...) DO UPDATE SET ... WHERE ...".
+    # Set to True for backends that support this feature.
+
+    supports_conditional_update = True
+
```

Comment:
We don't need to copy the comment here, it's in the base backend.

As before, let's call this `supports_update_conflicts_where` and move it up adjacent to `supports_update_conflicts`.